test/multi_node_test_2.py

```diff
@@ -58,7 +58,15 @@ def test_cluster():
     assert node2.all_processes_alive()
     g.remove_node(node2)
     g.remove_node(node)
-    assert not any(node.any_processes_alive() for node in g.list_all_nodes())
+    assert not any(n.any_processes_alive() for n in [node, node2])
+
+
+def test_shutdown():
```

Comment:
What is this intended to test? It seems redundant with the above test. Also, it doesn't hit the `atexit` code path right? Because the Python interpreter is not exiting.

Reply:
Ok, I see. It's not testing the `atexit` stuff though, right?

Reply:
No; it's just an additional test piggy-backing on this approval..